### PR TITLE
changed STATIC_URL and MEDIA_URL to fix errors with Compress

### PR DESCRIPTION
**Pull request by narnikgamarnikus**
[//]: # (Thank you for helping us out: your efforts mean great deal to the project and the community as a whole!)

[//]: # (Before you proceed:)

[//]: # (1. Make sure to add yourself to `CONTRIBUTORS.rst` through this PR provided you're contributing here for the first time)
[//]: # (2. Don't forget to update the `docs/` presuming others would benefit from a concise description of whatever that you're proposing)

## Description

[//]: # (What's it you're proposing?)

## Rationale

[//]: # (Why does the project need that?)

## Use case(s) / visualization(s)

[//]: # ("Better to see something once than to hear about it a thousand times.")

### CODE REVIEW

**Comment by webyneter** [comment]
Any explanation why this works and why the former wasn't working?

**Comment by narnikgamarnikus** [comment]
Django compress searches for files at STATIC_URL = f'https: // {AWS_STORAGE_BUCKET_NAME} .s3.amazonaws.com / static / ', but the default path was specified f'https: //s3.amazonaws.com/ {AWS_STORAGE_BUCKET_NAME} / static /

I deployed the project with these settings to the staging server and I managed to solve the problem this way.

File: {{cookiecutter.project_slug}}/.dockerignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 !.coveragerc
 !.env
 !.pylintrc
+node_modules/
```
Comment (webyneter) [inline {{cookiecutter.project_slug}}/.dockerignore]: This must be made conditional depending on presense or absense of `node_modules/`, for instance, this is the case when `cookiecutter.js_task_runner == "y"`. Other cases need to be thought over as well.

File: {{cookiecutter.project_slug}}/config/settings/production.py [hunk 2]
```diff
@@ -87,22 +87,22 @@
 STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
 {%- else %}
 STATICFILES_STORAGE = 'config.settings.production.StaticRootS3BotoStorage'
-STATIC_URL = f'https://s3.amazonaws.com/{AWS_STORAGE_BUCKET_NAME}/static/'
+STATIC_URL = f'https://{AWS_STORAGE_BUCKET_NAME}.s3.amazonaws.com/static/'
```
Comment (webyneter) [inline {{cookiecutter.project_slug}}/config/settings/production.py]: S3 buckets are reachable via subdomain (like you've got here) and `/` path notation alike. Therefore, the variant you propose is more generic which is a definitive plus.

File: hooks/post_gen_project.py [hunk 3]
```diff
@@ -310,21 +310,6 @@ def main():
...
-            WARNING
-            + "Docker and {} JS task runner ".format(
-                "{{ cookiecutter.js_task_runner }}".lower().capitalize()
-            )
-            + "working together not supported yet. "
-              "You can continue using the generated project like you "
-              "normally would, however you would need to add a JS "
-              "task runner service to your Docker Compose configuration "
-              "manually." + TERMINATOR
-        )
```
Comment (webyneter) [inline hooks/post_gen_project.py]: This is a change of different nature -- make sure to open another PR and address it there.

**Comment by narnikgamarnikus** [comment]
Take this PR.
I still have a few committees to improve.

**Comment by browniebroke** [comment]
Someone submitted another pull request so I want ahead and merged it. Thanks for your help.
